app/Models/Thread.php

```diff
@@ -212,4 +214,13 @@ public function replyAbleSubject(): string
     {
         return $this->subject();
     }
+
+    public function toSearchableArray(): array
+    {
+        return [
+            'subject' => $this->subject(),
+            'body' => $this->body(),
+            'url' => route('thread', $this->slug()),
```

Comment:
Maybe it's best to index the slug and ID so we don't hardcode the url here. If we ever need to change the url structure we'll avoid re-indexing everything.